### PR TITLE
Show in-game programming language

### PR DESCRIPTION
**Pull request by differentmatt**
Also removing ‘Code Saved’ status text, which was hidden by css bug
anyway.

### CODE REVIEW

**Comment by nwinter** [comment]
The "Code Saved" text isn't hidden by a CSS bug...? I use this every day.

**Comment by differentmatt** [comment]
Yea, looks like the CSS bug was of my own doing.

I don't think this is useful for students as is.  If we need to communicate this to users, we should revisit later.

File: app/styles/play/level/tome/spell.sass [hunk 1]
```diff
@@ -28,18 +28,27 @@
...
+    z-index: 4
+    font-family: "Open Sans Condensed", "Helvetica Neue", Helvetica, Arial, sans-serif
+    font-size: 14px
+    padding: 0px 10px
+    .programming-language-label
+      color: #6E654B
+      font-weight: bold
+      text-transform: uppercase
+    .programming-language
+      color: #1300BE
```
Comment (nwinter) [inline app/styles/play/level/tome/spell.sass]: This makes it look like a link, but it isn't a link. Can we stick with the same font and uppercase transform as on the label?

**Comment by nwinter** [comment]
Sure, it doesn't need to be displayed to users. I need something like it as an admin, tool, though. Can we keep it and make it subtle/fugly/left-aligned only when player is an admin or not in production?

**Comment by differentmatt** [comment]
Yup, will put saved status back for admins on the left.

File: app/styles/play/level/tome/spell.sass (hunk 1 above)
Comment (differentmatt) [inline app/styles/play/level/tome/spell.sass]: Good point.  Styling from @ryang217, so I'll sync up with her about making it less linkish.

**Pull request closed without merging**